### ISSUE TITLE
onPress on StackedBarChart

### ISSUE THREAD

**Issue by nthegedus** [JesperLekland/react-native-svg-charts#104]
### What is the problem?
Trying to use `onPress` with StackedBarChart

### When does it happen?
I'm trying to use onPress as same way the BarChart, but it crashes when try to render the graph.
### What platform?

* [X] iOS 
* [X] Android 

React Native version: 0.50.4

### Code to reproduce

```javascript
const barData =
      (this.props.data.totalPerAcquirePerDay).map((value, index) => ({
        value,
        style: {
          marginLeft: 40
        },
        svg: {
          onPress: () => { this._itemSelected(index) },
        },
      }))
....

<StackedBarChart
                keys={keys}
                colors={colors}
                data={barData}
                yAccessor={({ item }) => item.value}
                contentInset={{ top: 20, bottom: 5 }}
                spacingInner={0.5}
                renderGrid={Grid.Horizontal}
/>

```
### Stacktrace
```
index.bundle:12701 Warning: Failed prop type: The prop `d` is marked as required in `AnimatedPath`, but its value is `null`.
    in AnimatedPath (at stacked-bar-chart.js:188)
```
### Notes
```onPress``` on StackedBarChart is possible? I know the differences between BarChart, and StackedBarChart. Maybe, if I'm right, the problem is the values and keys that StackedBarChart need to map to render graph.

There's a way to use onPress on StackedBarChart?

**Comment by JesperLekland** [comment]
Hi @nthegedus! 

The `StackedBarChart` have not been updated to support the `svg` data property, i.e it does not support onPress at the moment. This is of course something that should be supported but not something I can promise that will be implemented especially soon.
You're very welcome to provide a PR with this functionality and I'll happily take a look at it.

**Comment by JesperLekland** [comment]
Included in `v5.1.x`

**Comment by JesperLekland** [comment]
See [this example](https://github.com/JesperLekland/react-native-svg-charts/blob/master/storybook/stories/bar-stack/with-on-press.js)
